modules/objdetect/perf/perf_qrcode_pipeline.cpp

```diff
@@ -114,7 +114,6 @@ PERF_TEST_P_(Perf_Objdetect_QRCode_Multi, decodeMulti)
         straight_barcode_sort.push_back(result[i].second);
     }
     SANITY_CHECK(decoded_info_sort);
-    SANITY_CHECK(straight_barcode_sort);
```

Comment:
Some problem in 4_qrcodes.png", 8_close_qrcodes.png
This QRs correctly decoded, but there is no bitwise equality in straight_barcode and golds. I think this is an optional check. A successful decode should mean that the test has passed.

Reply:
agreed